boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -110,10 +110,12 @@
 };
 
 &cpuapp_cpurad_ipc {
-	compatible = "zephyr,ipc-icmsg-me-initiator";
+	compatible = "zephyr,ipc-icbmsg";
```

Comment:
nit: Now that both sides use the same compatible, it can be moved to `nrf54h20dk_nrf54h20-ipc_conf.dtsi`

Reply:
Sounds good to me. Done